ios/Exponent/Versioned/Modules/Api/EXContacts.m

```diff
@@ -70,31 +68,60 @@ - (void)_getContactsWithPermissionGrantedAsync: (NSArray *)fields addressBook:(A
     ABRecordRef person = CFArrayGetValueAtIndex(allPeople, index);
     contact[@"id"] = @(ABRecordGetRecordID(person));
     
-    NSString *firstName = (__bridge_transfer NSString *)(ABRecordCopyValue(person, kABPersonFirstNameProperty));
-    NSString *lastName = (__bridge_transfer NSString *)(ABRecordCopyValue(person, kABPersonLastNameProperty));
-    NSString *name = [self _assembleDisplayNameFromFirstName:firstName lastName:lastName];
-    if (name) {
-      contact[@"name"] = name;
```

Comment:
Not sure if we need to check for value, AFAIK passing `null` should work via bridge.